File: C#/백준/Bronze/31654. Adding Trouble/Adding Trouble.cs
using System;

namespace q{
    class w{
        static void Main(){
            string[] aa = Console.ReadLine().Split();
            int sum=0;
            for (int i =0; i<3; i++){
                if(i<2){
                    sum+=int.Parse(aa[i]);
                }
                else{
                    if(int.Parse(aa[2])==sum){
                        Console.Write("correct!");
                    }
                    else{
                        Console.Write("wrong!");
                    }
                }
            }
        }
    }
}

File: Java/백준/Bronze/31654. Adding Trouble/Adding Trouble.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt(); int b = in.nextInt();
        if(in.nextInt()==a+b){
            System.out.println("correct!");
        }
        else{
            System.out.println("wrong!");
        }
    }
}